src/main/sensors/battery.c

```diff
@@ -214,15 +214,12 @@ void updateCurrentMeter(int32_t lastUpdateAt, rxConfig_t *rxConfig, uint16_t dea
     mAhDrawn = mAhdrawnRaw / (3600 * 100);
 }
 
-fix12_t calculateVbatPidCompensation(void) {
-    fix12_t batteryScaler;
+float calculateVbatPidCompensation(void) {
+    float batteryScaler =  1.0f;
     if (feature(FEATURE_VBAT) && batteryCellCount > 1) {
-        uint16_t maxCalculatedVoltage = batteryConfig->vbatmaxcellvoltage * batteryCellCount;
-        batteryScaler = qConstruct(maxCalculatedVoltage, constrain(vbat, maxCalculatedVoltage - batteryConfig->vbatmaxcellvoltage, maxCalculatedVoltage));
-    } else {
-        batteryScaler = Q12;
+        // Up to 25% PID gain. Should be fine for 4,2to 3,3 difference
```

Comment:
Yeah battery stuff was additional. More less some revert from long time ago. I will have a better look at it.
Its indeed up to 33%